### ISSUE TITLE
Are you trying to mount a directory onto a file (or vice-versa)?

### ISSUE THREAD

**Issue by eleaner** [bitvora/haven#15]
after upgrade to the latest version I suddenly get:

`Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/data/appdata/haven/app" to rootfs at "/app/haven": mount /data/appdata/haven/app:/app/haven (via /proc/self/fd/6), flags: 0x5400: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type`

**Comment by eleaner** [comment]
docker-compose
```

services:
  haven:
    container_name: haven
    build:
      context: ./haven
      dockerfile: Dockerfile
    env_file:
      - .env.haven
    volumes:
      - "./appdata/haven/db:/app/db"
      - "./appdata/haven/app:/app/haven"
    ports:
      - 3355:3355
    user: "${DOCKER_UID:-1000}:${DOCKER_GID:-1000}"
    networks:
      net:
    restart: unless-stopped
```

**Comment by barrydeen** [comment]
taking a look. i hate docker.

**Comment by barrydeen** [comment]
Should be fixed now in #20
